src/components/PostPageDetail/index.js

```diff
@@ -697,23 +698,23 @@ const styles = StyleSheet.create({
   textDesc: {
     fontFamily: fonts.inter[400],
     fontSize: 16,
-    color: '#000'
+    color: COLORS.black
   },
   more: {
-    color: '#0e24b3',
+    color: COLORS.blueZaffre,
     fontFamily: fonts.inter[400],
     fontSize: 14
   },
   content: {
     width,
-    shadowColor: 'rgba(0,0,0,0.5)',
+    shadowColor: COLORS.black50,
     shadowOffset: {
       width: 0,
       height: 1
     },
     shadowOpacity: 0.5,
-    backgroundColor: 'white',
-    borderBottomColor: '#C4C4C4',
+    backgroundColor: COLORS.white,
+    borderBottomColor: COLORS.balance_gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is beneficial as it replaces hardcoded color values with centralized color constants, improving maintainability and consistency of the codebase.
```diff
- color: '#000'
+ color: COLORS.black

- color: '#0e24b3',
+ color: COLORS.blueZaffre,

- shadowColor: 'rgba(0,0,0,0.5)',
+ shadowColor: COLORS.black50,

- backgroundColor: 'white',
+ backgroundColor: COLORS.white,

- borderBottomColor: '#C4C4C4',
+ borderBottomColor: COLORS.balance_gray,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->